spec/controllers/step_by_step_pages_controller_spec.rb

```diff
@@ -31,29 +31,48 @@
   end
 
   describe "#publish" do
-    it "sets the edition number of the change notes" do
-      stub_user.permissions << "GDS Editor"
+    context 'generates an internal change note' do
+      it "for major update" do
```

Comment:
Generally "it" plus the test name should read as a sentence. So, "it for major update" doesn't mean anything, whereas "it generates an internal note" does.

The context and test names could be switched. So the context could be "major updates" and the test name could be `it "generates an internal change note"` and "minor updates" could be another context.